lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -23,12 +23,14 @@ states:
   ALTERNATE_DOC_PASSPORT:
     events:
       next:
-        targetState: MITIGATION_04_DL_NO_MATCH_PAGE
+        targetState: WEB_DL_OR_PASSPORT
+        targetEntryEvent: alternate-doc-invalid-dl
```

Comment:
Overriding the entry event like this makes the journey engine a bit clunky, but the alternative seemed worse in the YAML (creating several copies of the nested journey with different entry events specified there).

Reply:
I think this is better - it's much clearer how to read the map. before you needed to understand that the event used to transition to the nested journey state was used again to dive into it. Now you can work it out without needing that knowledge.